presto-spi/src/main/java/io/prestosql/spi/security/ConnectorIdentity.java

```diff
@@ -14,21 +14,30 @@
 package io.prestosql.spi.security;
 
 import java.security.Principal;
+import java.util.Map;
 import java.util.Optional;
 
+import static java.util.Collections.emptyMap;
 import static java.util.Objects.requireNonNull;
 
 public class ConnectorIdentity
 {
     private final String user;
     private final Optional<Principal> principal;
     private final Optional<SelectedRole> role;
+    private final Map<String, String> credentials;
```

Comment:
please update `toString`

Reply:
I discussed with @dain and he wants to call these `extraCredentials` to make it clear that these are not the primary credentials for the request (which are not available after the authentication phase).

Also, please update the commit title

> Add credentials in Identity and ConnectorIdentity